internal/node/node.bzl

```diff
@@ -259,6 +265,11 @@ _NODEJS_EXECUTABLE_ATTRS = {
         Note, this can lead to different outputs produced by this rule.""",
         default = [],
     ),
+    "coverage": attr.bool(
+        doc = """Weather to collect code coverage information
```

Comment:
feels like something that should be requested for an execution, not configured on the target. did we ever find a way to sense whether we are running under `bazel coverage`?

Reply:
Agreed, there are some environment vars that are set under `bazel coverage`, looking through the links here https://github.com/bazelbuild/rules_nodejs/issues/638